frontend/asfui/authentication/views.py

```diff
@@ -30,9 +30,8 @@ def login_view(request):
             else:    
                 msg = 'Invalid credentials'    
```

Comment:
Please consider adding some error loggers here.

Reply:
Error will be displayed. so not required.